yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/RemoteProfileService.java

```diff
@@ -39,20 +36,16 @@
 
     public static RemoteProfileService newInstance() {
         return new RemoteProfileService(
-                JsonResourceFetcher.newInstance(),
                 new PathFactory(),
-                new HeadersFactory(),
-                SignedMessageFactory.newInstance());
+                SignedRequestBuilder.newInstance()
+        );
     }
 
-    RemoteProfileService(ResourceFetcher resourceFetcher,
-                         PathFactory profilePathFactory,
-                         HeadersFactory headersFactory,
-                         SignedMessageFactory signedMessageFactory) {
-        this.resourceFetcher = resourceFetcher;
+    RemoteProfileService(PathFactory profilePathFactory,
+                         SignedRequestBuilder signedRequestBuilder) {
         this.pathFactory = profilePathFactory;
-        this.headersFactory = headersFactory;
-        this.signedMessageFactory = signedMessageFactory;
+        this.signedRequestBuilder = signedRequestBuilder;
+
         apiUrl = System.getProperty(PROPERTY_YOTI_API_URL, DEFAULT_YOTI_API_URL);
     }
```

Comment:
The only way I can see this being non-breaking is to overload with the new constructor, deprecate this one, then add `this.signedRequestBuilder = SignedRequestBuilder.newInstance()` to this constructor